### PR TITLE
v32

### PR DESCRIPTION
**Pull request by chimosky**

### CODE REVIEW

File: keyboard.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
 HAND_YOFFSET = -15
 
 # Unicode symbol for the paragraph key.
-PARAGRAPH_CODE = '\xb6'
+PARAGRAPH_CODE = u'\xb6'
```
Comment (quozl) [inline keyboard.py]: What's this for?  It has no effect.

File: lessonscreen.py [hunk 2]
```diff
@@ -31,7 +31,7 @@
 import keyboard, medalscreen
 
 # Paragraph symbol unicode character.
-PARAGRAPH_CODE = '\xb6'
+PARAGRAPH_CODE = u'\xb6'
```
Comment (quozl) [inline lessonscreen.py]: And here.

File: keyboard.py (hunk 1 above)
Comment (chimosky) [reply keyboard.py]: 7f38e23  explains it.

**Comment by quozl** [comment]
Thanks.  Let's get this fixed in the toolkit.  https://github.com/sugarlabs/sugar-toolkit-gtk3/issues/440

**Pull request merged**